MoneySplitter.Win10/Converters/BankCardConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Windows.UI.Xaml.Data;
+
+namespace MoneySplitter.Win10.Converters
+{
+    public class BankCardConverter : IValueConverter
+    {
+        object IValueConverter.Convert(object value, Type targetType, object parameter, string language)
```

Comment:
better not use Interface definer here, only Convert(...

Reply:
fixed